.changeset/light-cheetahs-arrive.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/vertexai-preview': patch
+---
+
+Add a new VertexAI error type
```

Comment:
This will end up (through a human filter) in the release notes so we want to try to make it as detailed as is reasonable. How about "Add a publicly exported `VertexAIError` class."

Reply:
Sounds good! Done